### PR TITLE
Add MpPreferenceAsr in HailMary Mode

### PR DESCRIPTION
**Pull request by ataumo**

### CODE REVIEW

File: Invoke-HardeningKitty.ps1 [hunk 1]
```diff
@@ -1589,6 +1589,45 @@
                     $MessageSeverity = "High"
                 }
 
+                Write-ResultEntry -Text $Message -SeverityLevel $MessageSeverity
+            }
+            
+                        #
```
Comment (0x6d69636b) [inline Invoke-HardeningKitty.ps1:1595]:
```suggestion
                        #

```

**Comment by 0x6d69636b** [comment]
Could you update the version number at the end as well? I use the current unix timestamp when editing: `date +%s`

**Comment by ataumo** [comment]
Okay but, how can I do that ? Actually, I don't understand what you want ? Which version are you talking about ?

**Comment by 0x6d69636b** [comment]
The variable $HardeningKittyVersion contains the current version, like `$HardeningKittyVersion = "0.7.0-1647540083"`. Every commit or pull request should change this version. You can run the command `date +%s` (on Linux or macOS) to get the current timestamp and replace the version string with it. For examples "0.7.0-1647540083" becomes "0.7.0-1647615799"

**Comment by ataumo** [comment]
Okay thank you, I think everything is good now !

**Comment by 0x6d69636b** [comment]
Thank you very much

**Pull request merged**